charts/kong/CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
   `ingressController.konnect.license.enabled` is set to `true` that enables
   the synchronization of license with Konnect.
   [#1331](https://github.com/Kong/charts/pull/1331)
+* Generate Gateway API RBAC policy rules by default.
```

Comment:
Should we mention that this configuration will only create gwapi's rbac regardless of whether gwapi exists in the cluster, but if gwapi exists in the cluster, this configuration has no effect and cannot prevent gwapi's rbac from being created?

Reply:
Good point, #1342. PTAL